docs/xls-33d-multi-purpose-tokens/reference/mptoken.md

```diff
@@ -44,8 +44,8 @@ The `MPTokenID` is the result of SHA512-Half of the following values, concatenat
 | LedgerEntryType   | number    | UInt16        | The value 0x007F, mapped to the string `MPToken`, indicates that this object describes an individual account's holding of an MPT. |
 | `Account`           | string    | AccountID     | The owner of the MPT. |
 | `MPTokenIssuanceID` | string    | UInt256       | The `MPTokenIssuance` identifier. |
-| `MPTAmount`         | string    | UInt64        | This value specifies a positive amount of tokens currently held by the owner. Valid values for this field are between 0x0 and 0xFFFFFFFFFFFFFFFF. |
-| `LockedAmount`      | string    | UInt64        | (Default) Specifies a positive amount of tokens that are currently held in a token holder's account but that are unavailable to be used by the token holder. Locked tokens might, for example, represent value currently being held in escrow, or value that is otherwise inaccessible to the token holder. The initial value is 0 in order to save space on the ledger for an empty MPT holding. |
+| `MPTAmount`         | string    | Base10        | This value specifies a positive amount of tokens currently held by the owner. Valid values for this field are between 0x0 and 18,446,744,073,709,551,615. |
```

Comment:
The internal type is still UInt64 (unsigned integer, 64 bits), but what's changing is the way it's written to JSON. Instead of being a hexadecimal string, it'll be a base-10 number (i.e. the typical way we're used to writing numbers) quoted as a string, [a format the XRPL already uses in other contexts like XRP amounts](https://xrpl.org/docs/references/protocol/data-types/currency-formats#string-numbers).

Also, the prefix `0x` implies hexadecimal, so you should write `0` instead of `0x0`, and the maximum value is probably best written as 2<sup>64</sup>-1 (tip: you can use `<sup>` for superscripts) although I'm not sure it's strictly necessary to explain the valid range in this case since the valid range is exactly the range of numbers that can be represented in a 64-bit unsigned int, which programmers should be able to figure out from "UInt64" if they need to.

Also, as a nitpick, zero isn't a "positive amount".

```suggestion
| `MPTAmount`         | String - Number   | UInt64        | This value specifies the amount of tokens currently held by the owner, as a quoted integer. |
```